handlers/chikka/chikka_test.go

```diff
@@ -68,13 +68,31 @@ var defaultSendTestCases = []ChannelSendTestCase{
 		SendPrep: setSendURL},
 	{Label: "Plain Reply",
 		Text: "Simple Message", URN: "tel:+63911231234",
-		Status:       "W",
-		ResponseToID: 5,
-		ResponseBody: "Success", ResponseStatus: 200,
+		Status:               "W",
+		ResponseToID:         5,
+		ResponseToExternalID: "external-id",
+		ResponseBody:         "Success", ResponseStatus: 200,
 		PostParams: map[string]string{
 			"message":       "Simple Message",
 			"message_type":  "REPLY",
-			"request_id":    "5",
+			"request_id":    "external-id",
+			"mobile_number": "63911231234",
+			"shortcode":     "2020",
+			"request_cost":  "FREE",
+			"client_id":     "Username",
+			"secret_key":    "Password",
+			"message_id":    "10",
+		},
+		SendPrep: setSendURL},
+	{Label: "Failed Reply use Send",
```

Comment:
Not checking the Status since all requests are mocked to return 400s so it might be misleading here we are checking we retried with SEND when REPLY failed